### ISSUE TITLE
Scraper does not download videos

### ISSUE THREAD

**Issue by TheMissingPort** [DIGITALCRIMINALS/UltimaScraper#1002]
Im running 7.6.1

Anyone having the same issues or has a fix for it?

Thanks

**Comment by neil-ui** [comment]
Yes, I'm having the same issue. Everything is up do date and it downloads photos fine but not the videos.
